src/collectd.conf.in

```diff
@@ -558,12 +558,42 @@
 
 #<Plugin filecount>
 #	<Directory "/path/to/dir">
+#		#Plugin "foo"
 #		Instance "foodir"
 #		Name "*.conf"
 #		MTime "-5m"
 #		Size "+10k"
 #		Recursive true
 #		IncludeHidden false
+#		#FilesSizeType "bytes"
+#		#FilesCountType "files"
+#		#TypeInstance "instance"
+#	</Directory>
+#	#Postfix mail queue size
```

Comment:
Such example configurations should either go in the `contrib/` directory or (preferably) in the wiki. Some plugins already have a "…/Config" subpage and the [FileCount plugin](https://collectd.org/wiki/index.php/Plugin:FileCount) could have one, too.